libs/blocks/faas/utils.js

```diff
@@ -26,12 +26,12 @@ export const getFaasHostSubDomain = (environment) => {
   if (faasEnv === 'qa') {
     return 'qa.';
   }
-  return 'dev.';
+  return 'qa.';
 };
 
 const base = miloLibs || codeRoot;
 export const faasHostUrl = `https://${getFaasHostSubDomain()}apps.enterprise.adobe.com`;
-const faasCurrentJS = `${faasHostUrl}/faas/service/jquery.faas-current.js`;
+const faasCurrentJS = base.includes('localhost') ? `${base}/deps/jquery.faas-current.js` : `${faasHostUrl}/jquery.faas-current.js`;
```

Comment:
Getting a 404 error in the console for `https://qa.apps.enterprise.adobe.com/jquery.faas-current.js `. Looks like you accidentally dropped the `/faas/service` part

Reply:
I am so sorry, I was doing multiple things at the moment and didn't do the proper test myself. This surely works, thanks for caching all the misses.

Reply:
No problem, looks good now